src/app/components/progress-indicator/index.html

```diff
@@ -19,26 +19,26 @@
       isOptional="true"
       defaultValue="Vertical"
     >
-      Specifies whether to display the progress indicator vertically or horizontally. For <a routerLink="/components/wizard">modal wizards</a>, use the horizontal display mode. For waterfall pages and passive progress indicators, use the vertical display mode.
+      Specifies whether to display the progress indicator vertically or horizontally. For <a routerLink="/components/wizard">modal wizards</a>, use the horizontal display mode. For waterfall pages and passive progress indicators, use the vertical display mode. This property accepts <stache-code>string</stache-code> values.
```

Comment:
Based on https://github.com/blackbaud/skyux-progress-indicator/blob/master/src/app/public/modules/progress-indicator/types/progress-indicator-mode.ts, I think the options here are `Vertical` and `Horizontal`, but they might be `0` and `1`. If the former, I'd add a new next-to-last sentence: "The valid options are `Vertical` and `Horizontal`." If the latter, I'd add: "The valid options are `0` for the vertical display mode and `1` for the horizontal display mode." ... This property is not in use in the demo, and I couldn't get the Plunker to work, so we need to ask a dev.

Reply:
Reached out to Alex for help on this, and he offered to review the PR for accuracy.

Reply:
Looks good. The link didn't work for me, so I'm fixing that and expanding the link text to include more than just the code snippet so that it will stand out more clearly as a link.